src/controller/SetUpCodePairer.cpp

```diff
@@ -335,6 +370,36 @@ void SetUpCodePairer::OnBLEDiscoveryError(CHIP_ERROR err)
 }
 #endif // CONFIG_NETWORK_LAYER_BLE
 
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+void SetUpCodePairer::OnDiscoveredDeviceOverWifiPAF()
+{
+    ChipLogProgress(Controller, "Discovered device to be commissioned over WiFiPAF, RemoteId: %lu", mRemoteId);
+
+    mWaitingForDiscovery[kWiFiPAFTransport] = false;
+    auto param                              = SetUpCodePairerParameters();
+    param.SetPeerAddress(Transport::PeerAddress(Transport::Type::kWiFiPAF, mRemoteId));
+    mDiscoveredParameters.emplace_front(param);
+    ConnectToDiscoveredDevice();
+}
+
+void SetUpCodePairer::OnWifiPAFDiscoveryError(CHIP_ERROR err)
+{
+    ChipLogError(Controller, "Commissioning discovery over WiFiPAF failed: %" CHIP_ERROR_FORMAT, err.Format());
+    mWaitingForDiscovery[kWiFiPAFTransport] = false;
+    LogErrorOnFailure(err);
```

Comment:
We already logged the error.  Why is this LogErrorOnFailure here?

Reply:
I will remove the redundant one.